src/components/NormalRightRow/index.jsx

```diff
@@ -30,9 +54,17 @@ function NormalRightRow({ schema, classes }) {
               component="div"
               variant="subtitle2"
               className={classes.line}>
-              {keyword}
-              {': '}
-              {`${schema[keyword]}`}
+              {typeof schema[keyword] === 'object' &&
+              !Array.isArray(schema[keyword]) ? (
+                <Tooltip title={createTooltipTitle(keyword)} arrow>
+                  <Typography component="span" variant="subtitle">
```

Comment:
I'm not sure `subtitle` is a valid variant. https://material-ui.com/api/typography/

Reply:
Oh, my mistake. Applied changes!